Src/GoogleApis.Tests/Apis/Upload/ResumableUploadTest.cs

```diff
@@ -756,7 +758,7 @@ public void TestChunkUpload_NotFound_KnownSize()
             // From that point the server works as expected, and received the last chunks successfully
             var payload = Encoding.UTF8.GetBytes(UploadTestData);
 
-            var handler = new MultipleChunksMessageHandler(knownSize, error, payload.Length, chunkSize);
+            var handler = new MultipleChunksMessageHandler(knownSize, error, payload.Length, chunkSize) { ErrorMessage = jsonError };

```

Comment:
Just want to make sure that the line's length is not bigger than 120.
Unfortunately, we still don't have any stylecop rule for that (or anything else... :( )